packages/react-devtools-shared/src/utils.js

```diff
@@ -87,6 +85,9 @@ const encodedStringCache: LRUCache<string, Array<number>> = new LRU({
   max: 1000,
 });
 
+// Previously, the type of `Context.Provider`.
+const LEGACY_REACT_PROVIDER_TYPE: symbol = Symbol.for('react.provider');
```

Comment:
Not sure if you have another way to deal with devtools needing to support older React runtimes? I just moved the otherwise deleted constant here…